lib/ansible/modules/cloud/podman/podman_container.py

```diff
@@ -0,0 +1,1350 @@
+    type: dict
+  label_file:
+    description:
+      - Read in a line delimited file of labels
+    type: str
+  log_opt:
+    description:
+      - Logging driver specific options. Used to set the path to the container log file.
+        For example log_opt "path=/var/log/container/mycontainer.json"
+    type: str
+    aliases:
+      - log_options
+  memory:
+    description:
+      - Memory limit (format 10k, where unit = b, k, m or g)
+    type: str
+  memory_reservation:
+    description:
+      - Memory soft limit (format 100m, where unit = b, k, m or g)
+    type: str
+  memory_swap:
+    description:
+      - A limit value equal to memory plus swap. Must be used with the -m (--memory) flag.
+        The swap LIMIT should always be larger than -m (--memory) value.
+        By default, the swap LIMIT will be set to double the value of --memory
+    type: str
+  memory_swappiness:
+    description:
+      - Tune a container's memory swappiness behavior. Accepts an integer between 0 and 100.
+    type: int
+  mount:
+    description:
+      - Attach a filesystem mount to the container. bind or tmpfs
+        For example mount "type=bind,source=/path/on/host,destination=/path/in/container"
+    type: str
+  network:
+    description:
+      - Set the Network mode for the container
+        * bridge create a network stack on the default bridge
+        * none no networking
+        * container:<name|id> reuse another container's network stack
+        * host use the podman host network stack.
+        * <network-name>|<network-id> connect to a user-defined network
+        * ns:<path> path to a network namespace to join
+        * slirp4netns use slirp4netns to create a user network stack.
+          This is the default for rootless containers
+    type: str
+    aliases:
+      - net
+  no_hosts:
+    description:
+      - Do not create /etc/hosts for the container
+    type: bool
+  oom_kill_disable:
+    description:
+      - Whether to disable OOM Killer for the container or not.
+    type: bool
+  oom_score_adj:
+    description:
+      - Tune the host's OOM preferences for containers (accepts -1000 to 1000)
+    type: int
+  pid:
+    description:
+      - Set the PID mode for the container
+    type: str
+  pids_limit:
+    description:
+      - Tune the container's pids limit. Set -1 to have unlimited pids for the container.
+    type: str
+  pod:
+    description:
+      - Run container in an existing pod.
+        If you want podman to make the pod for you, preference the pod name with "new:"
+    type: str
+  privileged:
+    description:
+      - Give extended privileges to this container. The default is false.
+    type: bool
+  publish:
+    description:
+      - Publish a container's port, or range of ports, to the host.
+        Format - ip:hostPort:containerPort | ip::containerPort | hostPort:containerPort | containerPort
+    type: list
+    aliases:
+      - ports
+      - published
+      - published_ports
```

Comment:
Please, at most one alias, even better none. Having a long list of aliases is really bad UX IMO.